tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
-        "experimentalDecorators": true
+        "experimentalDecorators": true,
+        "module": "ESNext"
     }
 }
```

Comment:
I there a reason we are targeting ESNext? Because remember, when you do ESNext, we might be targeting features that TypeScript already compiles to, but Deno doesn't understand.